src/web/browser/ga/ga.ts

```diff
@@ -174,3 +174,11 @@ export const sendPageView = (): void => {
 
     trackLCP(send);
 };
+
+export const trackNonClickInteraction = (actionName: string): void => {
+    const send = `${tracker.name}.send`;
+
+    window.ga(send, 'event', 'Interaction', actionName, {
```

Comment:
Do you need a safety check that the `ga` function exists on `window`?

Reply:
I took a look at the the other functions in this module such as `sendPageView` and they're not checking, but I appreciate a check will provide an extra layer of security so will add one.

Reply:
depends on your idea of safety, I know we differ on this Paul, but I'd rather fail loudly than silently lose tracking.  The more bad data we accumulate, the harder it is to get out of.

Reply:
We could throw an error in the event it's not present, as we do here: https://github.com/guardian/dotcom-rendering/blob/master/src/web/browser/ophan/ophan.ts#L95, should then be captured and reported in `sentry`.

Reply:
In fact there's a `reportError` function exported by the `sentry` module that can be used rather than throwing a `new Error`: https://github.com/guardian/dotcom-rendering/blob/e9741e59c9120198fc1fc76ebefeeb3d5427c0de/src/web/browser/sentry/sentry.tsx#L58

Reply:
I'm now reporting any instances where `window.ga` is undefined with `reportError`.